install-dev/upgrade/php/migrate_tabs_17.php

```diff
@@ -104,6 +104,12 @@ public function createEntityTab($identifier, array $data, array $data_lang): voi
         if (isset($data['enabled'])) {
             unset($data['enabled']);
         }
+        if (isset($data['wording'])) {
```

Comment:
This would be worth a comment at least
Also I understand that you need to remove this to upgrade from 1.6 to 1.7

But doesn't that also mean that it won't work for more advanced versions that are updated? unless the `XmlLoader1700` class is only used during a step between 1.6 and 1.7.0.0? So the data should be correctly integrated/updated in a later step?

Reply:
Data will be correctly upgraded from 1.6 to 1.7.0.0, the problem occurs only with the latest 1.7.8.0 because both keys have been added recently